src/coreComponents/mesh/mpiCommunications/SpatialPartition.hpp

```diff
@@ -45,8 +45,33 @@ class SpatialPartition : public PartitionBase
 
   int getColor() override;
 
-  /// number of partitions
-  array1d< int > m_Partitions;
+  /**
+   * @brief Get the metis neighbors indices, const version. @see DomainPartition#m_metisNeighborList
+   * @return Container of global indices.
+   */
+  std::set< int > const & getMetisNeighborList() const
+  {
+    return m_metisNeighborList;
+  }
+
+  /**
+   * @brief Sets the list of metis neighbor list.
+   * @param metisNeighborList A reference to the Metis neighbor list.
+   */
+  void setMetisNeighborList( std::vector< int > const & metisNeighborList )
+  {
+    m_metisNeighborList.insert( metisNeighborList.cbegin(), metisNeighborList.cend() );
```

Comment:
```suggestion
    m_metisNeighborList.clear();
    m_metisNeighborList.insert( metisNeighborList.cbegin(), metisNeighborList.cend() );
```
Otherwise it's not a `set`, it's an insert. Maybe you want it to be an insert?